common/metrics/tags.go

```diff
@@ -30,7 +30,7 @@ import (
 	"strings"
 
 	enumspb "go.temporal.io/api/enums/v1"
-	serverenumbspb "go.temporal.io/server/api/enums/v1"
+	enumsspb "go.temporal.io/server/api/enums/v1"
```

Comment:
Not loving this change, it's hard to spot the difference between the 2 imports.

Reply:
If you want to propose a change to this convention across the whole repo, start a thread in the slack channel. Before that happens, let's please keep it consistent?